awx/ui_next/src/components/CodeEditor/CodeEditor.jsx

```diff
@@ -11,6 +13,8 @@ import { t } from '@lingui/macro';
 import styled from 'styled-components';
 import debounce from '../../util/debounce';
 
+config.set('loadWorkerFromBlob', false);
+
```

Comment:
For more context, see:

- https://github.com/ansible/awx/issues/9817
- [worker_client.js#L47](https://github.com/ajaxorg/ace/blob/master/lib/ace/worker/worker_client.js#L47)
- https://github.com/ajaxorg/ace/issues/3846
- https://github.com/ajaxorg/ace/pull/3862/files